src/commands/trace/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+import {AxiosPromise, AxiosResponse} from 'axios'
+
+export interface CustomIDs {
+  id: string
+  parent_id?: string
+}
+
+export interface Payload {
+  command: string
+  custom: CustomIDs
```

Comment:
are we using `CustomIDs` somewhere else? If not maybe just put it in `Payload` ?